### ISSUE TITLE
Mediator for the whole multitenant agent - get default-mediator-id Unauthorized

### ISSUE THREAD

**Issue by mkempa** [hyperledger/aries-cloudagent-python#1165]
Hi,

I have a mediator for the whole multitenant agent, in which there are no subwallets yet.
The first start of the multitenant agent with --mediator-invitation was successful and mediation was granted.
Subsequent starts of the multitenant agent require the --default-mediator-id parameter.

However, GET **{mutlitenantAdminAPI}/mediation/requests** returns 401: Unauthorized, therefore I am unable to find out the mediation id to put as the default mediator id.

I am aware of using the Bearer token for the subwallet, but I want to do it for the whole multitenant (the base wallet).

Could you please help me on how to do it?
Thanks

**Comment by TimoGlastra** [comment]
Hmm interesting. The admin API is blocking most api calls for the base wallet in multi-tenancy. Maybe we should make an exception for this specific endpoint?

**Comment by mkempa** [comment]
I found a workaround. To be clear, I have persisted wallets in postgres (for both mediator agent and edge agent).

1. Start mediator (with --open-mediation)
2. Start edge agent without --multitenant and --multitenant-admin params. It is now possible to include the --mediator-invitation param to automate the connection and mediation, or connect the agents manually via the REST calls.
3. GET {edgeAgentAdminAPI}/mediation/requests - find and save the "mediation_id'
4. Shut down the edge agent
5. Start up the edge agent again: 
 - WITH --multitenant and --multitenant-admin params
 - WITH --default-mediator-id <mediation_id from step 3>
 - WITHOUT --mediator-invitation

Now everything runs, the default mediator is set. I also created a sub-wallet, created a connection invitation on this sub-wallet, and indeed the routing key was present.
